### ISSUE TITLE
Training UCF-101 from scratch

### ISSUE THREAD

**Issue by ardasnck** [chuckcho/c3d-tensorflow2#1]
Thank you very much for your contribution.

Have you ever tried UCF101 training from scratch(5conv + 3fcs) on tensorflow instead of finetuning as it is done in original C3D implementation or in your “video-caffe” repository? 

It would be great if you can provide some information (graph,loss,top1-accuracy) like you did in "video-caffe".

Thank you very much.

**Comment by chuckcho** [comment]
haven't done that yet. conversion from caffe to keras/tensorflow has some issue as of now. will update it soon.

**Comment by ardasnck** [comment]
What is the issue exactly? If you can describe it more, i can check it out. I already implemented the conversion from caffe to tensorflow successfully for feedforward.

**Comment by ardasnck** [comment]
Any progress on your tensorflow implementation?

**Comment by chuckcho** [comment]
I'm closing this repo as it's obsolete. https://github.com/axon-research/c3d-keras would be more relevant.
